### DISCUSSION TITLE
HOW TO CONTACT DEVELOPER ?

### DISCUSSION THREAD

**Discussion by raveen2k3** [CosmicPredator/anikimiapi#5] [General]
HOW CAN WE CONTACT , AVAILABLE IN SOCIAL MEDIA LIKE TELEGRAM ?

**Comment by sudo-nautilus** [comment]
You can [mail](mailto:kumarbarani61@gmail.com) that person

**Comment by CosmicPredator** [comment]
You can contact me at Telegram at @cosmicpredator

**Comment by Praveensenpai** [comment]
this doesn't work now...

now it only gives links as None
